src/components/Text/__tests__/text-tests.jest.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { fireEvent, render, cleanup } from "@testing-library/react";
+import { act } from "@testing-library/react-hooks";
+import Text from "../Text";
+
+const renderComponent = props => {
+  return render(<Text {...props} />);
+};
+
+describe("Text tests", () => {
```

Comment:
missing tests. I'd add here everything that's related to an interaction with the component

Reply:
No interactions in this component. only hover and display tooltip. This test will be implemented with storybook interaction in next pr (there is a chance it will be problematic to implement it )